sql/sql_show.cc

```diff
@@ -8838,6 +8881,17 @@ ST_FIELD_INFO authinfo_fields_info[]=
   {0, 0, MYSQL_TYPE_STRING, 0, 0, 0, SKIP_OPEN_TABLE}
 };
 
+ST_FIELD_INFO slave_db_load_fields_info[]=
+{
+  {"DB", NAME_LEN, MYSQL_TYPE_STRING, 0, 0, "Database",
+   SKIP_OPEN_TABLE},
+  {"WORKER", 21, MYSQL_TYPE_LONGLONG, 0, MY_I_S_UNSIGNED, "Worker ID",
+   SKIP_OPEN_TABLE},
+  {"DB_LOAD", 21, MYSQL_TYPE_LONGLONG, 0, MY_I_S_UNSIGNED, "DB Load",
```

Comment:
This denotes the load on the DB since the last checkpoint, not the total load since the server started.